04_numeronym/willshen8/numeronyms.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
+	returnNumeronyms := []string{}
+	for _, val := range vals {
+		returnNumeronyms = append(returnNumeronyms, convertStringToNumeronym(val))
+	}
+	return returnNumeronyms
+}
+
+func convertStringToNumeronym(input string) string {
+	regEx := regexp.MustCompile(`[^a-zA-Z]+`)
+	formattedInput := regEx.ReplaceAllString(input, "")
```

Comment:
why not make it work with foreign languages too? (just curious)

Reply:
Haha, great point. Certainly not for Asian languages, perhaps European languages with English alphabets?